manager/controllers/systems_advisories_view.go

```diff
@@ -0,0 +1,110 @@
+package controllers
+
+import (
+	"app/base/database"
+	"app/manager/middlewares"
+	"github.com/gin-gonic/gin"
+	"github.com/jinzhu/gorm"
+	"net/http"
+)
+
+type AdvisoryName string
```

Comment:
why do you use these typedefs?

Reply:
Well, it was getting hard to read the code.

Reply:
For me it's even harder now. It looks like usage of new structs but it's only string. But if you like it ...